save-plugins/warn-plugin/src/commonMain/kotlin/com/saveourtool/save/plugin/warn/WarnPlugin.kt

```diff
@@ -151,6 +152,7 @@ class WarnPlugin(
             warnMissingExpectedWarnings(warnPluginConfig, generalConfig, originalPaths)
         }
 
+        // FixMe: When actualWarningsFileName is not null, we can skip this execution
```

Comment:
please try to catch this fixme in this PR too

Reply:
Fixed